dashboard/src/pages/TreeDetails/Tabs/LogSheet.tsx

```diff
@@ -122,7 +122,7 @@ export const LogSheet = ({
     { enabled: !!logUrl },
   );
   return (
-    <SheetContent className="flex w-[25rem] flex-col sm:w-full sm:max-w-[44rem]">
+    <SheetContent className="flex flex-col sm:w-full sm:max-w-[clamp(44rem,75rem,100vw)]">
```

Comment:
`rem` means the font-size of the root element, so it is not the proper unit to use in this case. It makes more sense to use `vw` for the preferred value (the second argument) and `px` for the minimum and maximum. The maximum should be the maximum width of the `SheetContent` that still leaves room for the left sidebar + the page content (at the background).